serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/evn python3
+
+liter_or_less = int(input("Enter the number of containers that holding 1 liter or less:"))
+more_than_liter = int(input("Enter the number of containers that holding more than 1 liter:"))
+total = (0.10 * liter_or_less) + (0.25 * more_than_liter)
```

Comment:
LW: What represents numbers 0.10 and 0.25. Use constant variables for these numbers.